llvm/lib/Target/AMDGPU/AMDGPUOclcReflect.cpp

```diff
@@ -79,6 +85,9 @@ PreservedAnalyses AMDGPUOclcReflectPass::run(Function &F,
     if (ReflectArg == "AMDGPU_OCLC_UNSAFE_INT_ATOMICS") {
       int ReflectVal = AMDGPUUnsafeIntAtomicsEnable ? 1 : 0;
       Call->replaceAllUsesWith(ConstantInt::get(Call->getType(), ReflectVal));
+    } else if (ReflectArg == "AMDGPU_OCLC_UNSAFE_FP_ATOMICS") {
+      int ReflectVal = AMDGPUUnsafeFPAtomicsEnable ? 1 : 0;
```

Comment:
nit: I know it's already there, but `bool` is directly convertible to `int` and is guaranteed to be 0 or 1, so this reads weird to me.

Reply:
You know what: it's fine; ignore me

Reply:
Fair enough, change made.